R/create_individual_file.R

```diff
@@ -794,54 +793,3 @@ join_slf_lookup_vars <- function(individual_file,
 
   return(individual_file)
 }
-# TODO Remove the client data from the individual Social Care extracts
-# and instead, use this function in the episode file to match on the client
-# data to all episodes.
-#' Join sc client variables onto individual file
-#'
-#' @description Match on sc client variables.
-#'
-#' @param individual_file the processed individual file
-#' @param year financial year.
-#' @param sc_client SC client lookup
-#' @param sc_demographics SC Demographic lookup
-join_sc_client <- function(
-    individual_file,
-    year,
-    sc_client = read_file(get_sc_client_lookup_path(year)),
-    sc_demographics = read_file(get_sc_demog_lookup_path(),
-      col_select = c("sending_location", "social_care_id", "chi")
-    )) {
-  # TODO Update the client lookup processing script to match
-  # on demographics there so the client lookup already has CHI.
-
-  # Match to demographics lookup to get CHI
-  join_client_demog <- sc_client %>%
-    dplyr::left_join(
-      sc_demographics %>%
-        dplyr::select("sending_location", "social_care_id", "chi"),
-      by = c("sending_location", "social_care_id")
-    ) %>%
-    dplyr::mutate(count_not_known = rowSums(dplyr::select(., all_of(
-      c(
-        "sc_living_alone",
-        "sc_support_from_unpaid_carer",
-        "sc_social_worker",
-        "sc_meals",
-        "sc_day_care"
-      )
-    )) == "Not Known")) %>%
-    dplyr::arrange(chi, count_not_known) %>%
-    dplyr::distinct(chi, .keep_all = TRUE)
```

Comment:
I can see where you have split this out, just wondering about this section if it is needed?

Reply:
No. `process_lookup_sc_client` did not use this.